csep/utils/readers.py

```diff
@@ -500,7 +500,7 @@ def ingv_emrcmt(fname):
     n_event = 0
     with open(fname) as file_:
         reader = csv.reader(file_)
-        
+        next(reader, None) # skip header
```

Comment:
does this properly handle the case where their isn't a header? should we add some logic here to read the first line and somehow determine if this is in fact a header line

Reply:
INGV now always give their rcmt catalogs with a header.
Unfortunately csv.reader does not handle this elegantly like pandas. We agreed before to use csv instead of pandas to minimize the dependencies.

Reply:
is there a possibility that someone would try to use an old-style catalog? if so, i think we should check for the existence of the header line. in the csep.utils.readers.csep_ascii() function i wrote a short function to check for this header line. maybe we can do something similar here.

Reply:
Modified!